tket/src/CMakeLists.txt

```diff
@@ -55,6 +55,9 @@ ELSEIF(APPLE)
     # set correct install_name
     set(CMAKE_INSTALL_NAME_DIR "@loader_path")
     set(CMAKE_BUILD_WITH_INSTALL_NAME_DIR ON)
+ELSEIF(UNIX)
+    set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
+    set(CMAKE_INSTALL_RPATH "\${ORIGIN}")
```

Comment:
This is not required for the test package? or the other tket modules?

Reply:
Not required for the test package because nothing else uses it. This is only (currently) necessary for libraries that are packaged with pytket.